### PR TITLE
Widget: Fix initialization of TextEditor

### LINKED ISSUE #27: Check TextEditor call
Check whether we need something similar to spyder-ide/spyder#12094

### PR DESCRIPTION
**Pull request by jitseniesen**
Fixes #27

### CODE REVIEW

File: spyder_memory_profiler/widgets/memoryprofiler.py [hunk 1]
```diff
@@ -196,15 +196,21 @@ def select_file(self):
...
 
     def show_errorlog(self):
         if self.error_output:
-            TextEditor(self.error_output, title=_("Memory profiler output"),
-                       readonly=True, size=(700, 500)).exec_()
+            editor = TextEditor(self.error_output,
+                                title=_("Memory profiler output"),
+                                readonly=True)
+            editor.show()
+            editor.exec_()
```
Comment (goanpeca) [inline spyder_memory_profiler/widgets/memoryprofiler.py:217]: Why not use just `editor.exec_()` I beleived that called show first?

Comment (jitseniesen) [reply spyder_memory_profiler/widgets/memoryprofiler.py:217]: Having just `editor.exec_()` does not work. I don't know exactly why, but it looks like that does not call `.show()`.

Comment (goanpeca) [reply spyder_memory_profiler/widgets/memoryprofiler.py:217]: Hmmm weird. Maybe adding a comment could help :-p ?

Comment (jitseniesen) [reply spyder_memory_profiler/widgets/memoryprofiler.py:217]: My explanation was incomplete. After spyder-ide/spyder#10976, `TextEditor` inherits from `BaseDialog` which overrides `QDialog`'s `.show()` to dynamically resize the dialog window based on the screen size. However, `.exec()` does not call the override and thus, while it does show the dialog window, it does not dynamically resize it.

Anyway, I added a comment referencing the issue that kind of explains this.

**Pull request merged**